core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJZipResolver.scala

```diff
@@ -37,17 +38,10 @@ object IntelliJZipResolver extends IntelliJResolver {
       .resolver(artifact)
   }
 
-  def fromConfig(config: DependenciesConfig.IntelliJ): Seq[DependencyResolver[IntelliJVersion]] = {
-    val official = Seq(
-      IntelliJZipResolver.community,
-      AlternativeIntelliJZipResolver.community,
-      NightlyIntelliJZipResolver.community
-    )
-    val fromConfig = config.repositories
-      .flatMap(pattern =>
-        if (Set("official", "default").contains(pattern.toLowerCase)) official
-        else Seq(IntelliJPatternResolver(pattern).community)
-      )
-    if (fromConfig.isEmpty) official else fromConfig
+  // this method enables downloading .zip IntelliJ even if `probe.intellij.version.ext` is ".dmg"
```

Comment:
Why is it needed? Is installing IJ from ".dmg" supported or not?

Reply:
Looks like installing from .dmg is not supported in reality. Created an issue: https://github.com/VirtusLab/ide-probe/issues/281
If we add the support for `.dmg` in the future, then it would be needed. Let me fix user-facing comments so that it will be obvious, that for now we do not support `.dmg` extensions.

Reply:
Fixed user-facing comments about .dmg extension

Reply:
Support for installing from `.dmg` should be available after merging https://github.com/VirtusLab/ide-probe/pull/283